api/src/auth/auth.module.ts

```diff
@@ -0,0 +1,26 @@
+import { Module } from '@nestjs/common';
+import { AuthService } from './auth.service';
+import { LocalStrategy } from './local.strategy';
+import { UsersModule } from '../users/users.module';
+import { PassportModule } from '@nestjs/passport';
+import { JwtModule } from '@nestjs/jwt';
+import { ConfigService, ConfigModule } from '@nestjs/config';
+import { JwtStrategy } from './jwt.strategy';
+
+@Module({
+  imports: [
+    UsersModule,
+    PassportModule,
+    JwtModule.registerAsync({
+      imports: [ConfigModule],
+      useFactory: (config: ConfigService) => ({
+        secret: config.get<string>('BOS_JWT_SECRET'),
```

Comment:
This will error if `BOS_JWT_SECRET` isn't set in `process.env. BOS_JWT_SECRET`. That means e2e tests in CI will fail until we add `BOS_JWT_SECRET` to the e2e job.

Reply:
🤔  We'll want to store this in a rotatable location like in a AWS secret store for production. We can manually create that then inject it into the lambda in the `serverless.yml`

Reply:
For CI, we can just pass in a hardcoded secret in the e2e job.